core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -406,12 +406,13 @@ class DAGSchedulerSuite extends SparkFunSuite with TempLocalSparkContext with Ti
     blockManagerMaster = spy(new MyBlockManagerMaster(sc.getConf))
     doNothing().when(blockManagerMaster).updateRDDBlockVisibility(any(), any())
     scheduler = new MyDAGScheduler(
+    scheduler = spy(new MyDAGScheduler(
```

Comment:
This should cause a compilation failure - remove the prev line ?

Reply:
My bad...there was a commit fixing the issue not pushed. Updated